README.md

```diff
@@ -12,21 +12,21 @@ We implement the Kubernetes-recommended node-level logging architecture. The Age
 
 You can also install the Agent to monitor other parts of the infrastructure, for example a hosted database service.  
 
-For more metrics and insight into your cluster, this chart can also install Kubernetes Explorer (in preview release). This enables a third,  [Openmetrics monitor](https://github.com/scalyr/scalyr-agent-2/tree/master/scalyr_agent/builtin_monitors). The standard installation has approximately 60 metrics; Kubernetes Explorer provides over 500 out of the box. Open source [metric exporters](https://prometheus.io/docs/instrumenting/exporters/) also let you easily collect metrics from applications running in your cluster.
+For more metrics and insight into your cluster, this chart can also install Kubernetes Explorer (in preview release). This enables a third,  [Openmetrics monitor](https://github.com/scalyr/scalyr-agent-2/tree/master/scalyr_agent/builtin_monitors). The standard installation has approximately 60 metrics; Kubernetes Explorer provides over 500 out of the box. You can easily collect more metrics from applications running in your cluster with open source [metric exporters](https://prometheus.io/docs/instrumenting/exporters/).
 
 
 ## Installation
 
 You must set some configuration options:
-- ``scalyr.apiKey``: Must be a "Log Write Access" API key. Log into your DataSet account. Select your account (email address), then select "Api Keys".
+- ``scalyr.apiKey``: Must be a "Log Write Access" API key. Log into DataSet and select your account (email address). Then select "Api Keys".
 - ``scalyr.k8s.clusterName``: You must set a name for your Kubernetes cluster, which shows in the UI.
-- By default data uploads to our US server. For EU customers, set `scalyr.server="eu.scalyr.com"`.
+- By default data uploads to our US server. For EU customers, set `scalyr.server=eu.scalyr.com`.
 
 
 To install:
 
 ```bash
-helm install <name of release> scalyr-agent --repo https://scalyr.github.io/helm-scalyr/ --set scalyr.apiKey="<your write logs api key>" --set scalyr.k8s.clusterName="<your-k8s-cluster-name>"
+helm install <name of release> scalyr-agent --repo https://scalyr.github.io/helm-scalyr/ --set scalyr.apiKey=<your write logs api key> --set scalyr.k8s.clusterName=<your-k8s-cluster-name>
```

Comment:
Were quotes intentionally removed? It can be a good idea to include them to prevent bash expansion / splitting in case the actual string value contains special characters (e.g. $, whitespace, etc.).

Reply:
@kami My bad. I tested the command yesterday (zsh) and got the `dquote>` prompt. I thought all double quotes were closed, so I then checked the Helm documentation and their examples do not use quotes. So I thought this is standard for Helm.  

I just tested this command with quotes again in zsh and bash, and all is well. I can add the quotes back if you want?